### DISCUSSION TITLE
Taskbar overlapping start menu

### DISCUSSION THREAD

**Discussion by kyrcy** [valinet/ExplorerPatcher#3528] [General]
With 22621.3880.66.4 on 24H2 (10.0.26100.1150) the taskbar (Windows 10 ExplorerPatcher) is overlapping the start menu (Windows 10). Is this the expected behavior (see screenshot)?
<img width="462" alt="overlapping" src="https://github.com/user-attachments/assets/17d74497-b74c-4d68-92da-50ce0745a144">

**Comment by Amrsatrio** [comment]
I see no problems in your screenshot.

**Reply by kyrcy** [reply]
EPSON folder is half hidden by the taskbar and to see anything below that scrolling down is needed. I can't remember if this is the behavior in Windows 10. Also, clicking in search box opens a Windows 11 style window. Shouldn't there be a choice for a Windows 10 style window as well?

**Reply by Amrsatrio** [reply]
It is the behavior in Windows 10, the start menu apps list is scrollable and does not overlap on the taskbar.

Also, the search menu cannot be restored to Windows 10 because the one in 11 uses a completely different system.

**Reply by pyrates999** [reply]
Yep, I'm on windows 10 and that is normal behavior with the start menu.  So not a bug with EP.

Please close this.
